### ISSUE TITLE
jackett.json in qBitorrent not located in qBitorrent folder 

### ISSUE THREAD

**Issue by Eyji** [qbittorrent/search-plugins#92]
The jackett.json needed when using the search engine in qBitorrent is not located in `%localappdata%\qBittorrent\nova3\engines\` as specified [here](https://github.com/qbittorrent/search-plugins/wiki/How-to-configure-Jackett-plugin). Instead, it is based in a python folder. Had to search it using [Everything](https://www.voidtools.com/). See capture below.

[Jackett.json location using Everything](https://user-images.githubusercontent.com/10639815/58221228-13685a00-7d12-11e9-93e1-102e9acf9aa9.png)

Creating the jackett.json file in the same folder as the jackett.py file would result in an api key missing error.

**Comment by ngosang** [comment]
I think you have an unusual installation of Python. It's working for most people.
Maybe this issue help others. Thank you for the report.

**Comment by Piccirello** [comment]
Certainly we should look into this more before just closing the issue?

**Comment by Eyji** [comment]
Indeed, I have installed python form the Microsoft Store on this machine, which will be more and more common in the future :)

**Comment by ngosang** [comment]
@Piccirello This repository only contains the plugins itself. The Python detection, the updater, etc are in the in the qBittorrent core repository. Most of the code is in C++ and I can't help, maybe you can take a look with @Chocobo1 

The weird thing in this issue is the plugins have been copied to a weird directory. This is handled by the updater code.
![image](https://user-images.githubusercontent.com/10577978/58353007-9baf4200-7e6d-11e9-93d4-26303a292b62.png)

There are other related issues related to python detection qbittorrent/qBittorrent#10658

**Comment by p060477** [comment]
hi i run qbittorrent portable 4.1.4 and i got jackett api key error...
i've jackett.jason in c users appdata local qbittorrent nova3 engines
i really do not have any phyton folder in
c users appdata local packages
i've phyton installed 3.6.6 vers in my win 10 home 64 vers 2004
how to solve?
thxs so much in adv
cheers

**Comment by p060477** [comment]
..any ideas...??
have you jackett search plugin in qbittorrent WORKING..??
..may be not at all..

**Comment by Joe-Rees** [comment]
I recently got a new PC and installed BitTorrent from scratch. However, I do not have jackett.json. I searched my entire system (three drives)and turned up empty. I uninstalled BitTorrent and reinstalled it. Still no jackett.json!
Is there a dowload step I am missing?

**Comment by ngosang** [comment]
File `jackett.json` is created after the fist search in Jackett (the first search creates the file and it's expected to fail).
The alternative is to create the file yourself => https://github.com/qbittorrent/search-plugins/wiki/How-to-configure-Jackett-plugin#configuration-file

**Comment by webmasterMeyers** [comment]
I am having this same issue - worked before fine.  However, recently I installed AWS EB CLI which did some fancy python stuff, and I assume that broke it.

I think this
>Creating the jackett.json file in the same folder as the jackett.py file would result in an api key missing error.  

is the problem I am having
